src/lcms_metab_metadata_generator.py

```diff
@@ -110,14 +110,22 @@ def __init__(
         raw_data_url: str,
         process_data_url: str,
         minting_config_creds: str = None,
+        workflow_version: str = None,
     ):
         super().__init__(
             metadata_file=metadata_file,
             database_dump_json_path=database_dump_json_path,
             raw_data_url=raw_data_url,
             process_data_url=process_data_url,
         )
-
+        # Set the workflow version, prioritizing user input, then fetching from the Git URL, and finally using a default.
+        self.workflow_version = (
+            workflow_version
+            or self.get_workflow_version(
+                workflow_version_git_url="https://github.com/microbiomedata/metaMS/blob/master/.bumpversion_metabolomics.cfg"
+            )
+            or "1.0.0"
```

Comment:
Let's leave a #TODO to remove this "or" statement once we have a proper config set up.